### PR TITLE
Dltp 1202

### PR DESCRIPTION
**Pull request by msuhovec**

### CODE REVIEW

File: tasks/do-fedora-only-content [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  # compute content MD5 checksum
+
+  checksum=$(md5sum $JOBPATH/$unique_file_label | cut -d' ' -f 1)
+
+  checksum_found=$(grep "$checksum" "$JOBPATH/content_ds" | wc -l) 
+
+  # if MD5 checksum not present in datastream, return error
+
+  if [ $checksum_found -eq 0 ]; then
```
[the remark this answers is not in the document]
Comment (msuhovec) [reply tasks/do-fedora-only-content]: The comparison of checksum with checksum found is done in line 50.

I can add another check to see if the checksum attribute exists in the datastream at all...

File: tasks/do-fedora-only-content [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    exit 1
+fi
+
+# Iterate through pids- use rof fedora_to_rof to retrieve from fedora and generate rof
+for pid in $(jq -r 'to_entries[] | [.key, .value] | @csv' ${pid_file}); do
+  fedora_id=`echo $pid | sed 's/"//g' | sed 's/und://' | awk 'BEGIN{FS=","}{ print $1}'`
+  bendo_id=`echo $pid | sed 's/"//g' | sed 's/und://' | awk 'BEGIN{FS=","}{ print $2}'`
+
+  file_label=$(jq --raw-output '.[]["content-meta"]["label"]' "$JOBPATH/metadata-${fedora_id}.rof")
+  unique_file_label="${fedora_id}-${file_label}"
```
[the remark this answers is not in the document]
Comment (msuhovec) [reply tasks/do-fedora-only-content]: At this point, the file label name will contain the wierd character, which is sanitized down the line in filename-normalize.

[the remark this answers is not in the document]
Comment (dbrower) [reply tasks/do-fedora-only-content]: ok. cool.

File: tasks/do-fedora-only-content (hunk 1 above)
[the remark this answers is not in the document]
Comment (dbrower) [reply tasks/do-fedora-only-content]: ah, yes, I now see the "comparison" happening in line 50.

**Pull request merged**